Resources/Prototypes/Nyanotrasen/Voice/speech_emotes.yml

```
@@ -2,7 +2,7 @@
 - type: emote
   id: Hiss
   category: Vocal
-  chatMessages: [ шипит ]
+  chatMessages: [ chat-emote-msg-hisses ]
```

yamllint

[error] too many spaces inside brackets

(brackets)